internal/v2/tls_config_store/BUILD

```diff
@@ -0,0 +1,31 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+package(
+  default_visibility = ["//internal/v2:__subpackages__"],
+)
+
+#TODO(rmehta19) : Investigate how to use embedsrcs with data in a top level directory. If possible, move example_cert_key under s2a-go/internal/v2
```

Comment:
micro-nit: I think there should be a space between "#" and "TODO". In general, I suggest running "go mod tidy" on your code to clean up minor formatting stuff like this.

It's on my TODO list to add another CI that checks if there are formatting errors but I haven't had the time to do so yet. :(

Reply:
Will do, thanks for the tip!